peps/converters.py

```diff
@@ -160,6 +165,21 @@ def convert_pep_page(pep_number, content):
     # Fix PEP links
     pep_content = BeautifulSoup(data['content'])
     body_links = pep_content.find_all("a")
+    # Fix hihglighting code
+    code_blocks = pep_content.find_all("pre", class_="code")
```

Comment:
Nitpick: I know we are already pretty inconsistent with this, but please use single quotes where possible.